Tasks/Common/clienttool-common/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "clienttool-common",
+  "version": "1.0.0",
+  "description": "Azure Pipelines client tool Tasks Common",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Microsoft Corporation",
+  "category": "Package",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@types/mocha": "5.2.6",
+    "@types/node": "10.12.9",
```

Comment:
You'll need to add the "adm-zip" dependency here so that that module actually gets included in the task. The version that you need to include should be 0.4.13, which should match the version in the package.json file at the root of the repo